src/components/ExpandingContainer/ExpandingContainer.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { FC, useState } from "react";
 
-import { containerTitleToPreselectMap } from "@/components/DownloadAndShareComponents/helpers/containerTitleToPreselectMap";
+import { containerTitleToPreselectMap } from "@/components/TeacherComponents/helpers/downloadAndShareHelpers/containerTitleToPreselectMap";
```

Comment:
feels a bit unpleasant for a generic component to import from a squad-specific component, should one or the other be moved?

Reply:
yes! thanks for spotting - I will update now

Reply:
I moved `ExpandingContainer` to `ArchivedComponents` aa it is not used anywhere